corehq/apps/locations/models.py

```diff
@@ -317,18 +317,18 @@ def filter_by_user_input(self, domain, user_input):
     def get_locations(self, location_ids):
         return self.filter(location_id__in=location_ids)
 
-    def get_locations_and_children(self, location_ids):
+    def get_locations_and_children(self, location_ids, **filters):
```

Comment:
Similar to my previous comment, adding `**filters` here adds unnecessary complexity and a small runtime overhead. The filters can be added to the returned query object with `.filter(**filters)` if needed.